examples/README.md

```diff
@@ -72,6 +72,14 @@ neurodocker generate [docker|singularity] --base=debian:stretch --pkg-manager=ap
   --install git vim
 ```
 
+by default --install uses --no-install-recommends to minimize container sizes. In few cases this can lead to unexpected behaviours and one can try to build a container without this option.
+
+```shell
```

Comment:
can you also add an example with `yum`? for example using `--install yum_opts='--debug' git vim`. just remember to use some base image that uses `yum` (e.g., `centos:7`) and set `--pkg-manager=yum`.